### PR TITLE
KFSPTS-8347 sae email

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
Moving the shared objects to a super class seems easy enough to do now, let me give it a try.  If it becomes bigger than that, maybe something for the tech backlog would make more sense.

**Comment by jhulslander** [comment]
I moved the shared information into EmailFileFixture.java.  This is ready for review again.

File: src/test/java/edu/cornell/kfs/concur/batch/service/impl/fixture/EmailFileFixture.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+public enum EmailFileFixture {
+    
+    EMPTY_FILE("src/test/resources/edu/cornell/kfs/concur/batch/service/impl/fixture/empty.txt", StringUtils.EMPTY),
+    SIMPLE_FILE("src/test/resources/edu/cornell/kfs/concur/batch/service/impl/fixture/simpleContents.txt", "This is the first line.\n\tThis is the second line indented.");
+    
+    public final String fullFilePath;
+    public final String fileContents;
+    
+    private EmailFileFixture(String fullFilePath, String filContents) {
```
Comment (cah292) [inline src/test/java/edu/cornell/kfs/concur/batch/service/impl/fixture/EmailFileFixture.java]: Minor typo in variable name.

**Comment by jhulslander** [comment]
I fixed the type, this is ready for review again

**Pull request merged**